data/mods/Medieval_Stuff/recipes.json

```diff
@@ -0,0 +1,890 @@
+[
+{
+        "type" : "recipe",
```

Comment:
If you're going to copy-paste a recipe all over the place, for the love of $deity, fix the formatting first.